ts/screens/preferences/ServiceDetailsScreen.tsx

```diff
@@ -222,23 +245,16 @@ class ServiceDetailsScreen extends React.Component<Props, State> {
                   value={this.state.uiEnabledChannels.inbox}
                   disabled={
                     profileEnabledChannels.inbox !==
-                    this.state.uiEnabledChannels.inbox
+                      this.state.uiEnabledChannels.inbox ||
+                    pot.isLoading(this.state.updateStatus)
```

Comment:
oh this is probably the issue - when `dispatchNewEnabledChannels` gets called, a `profileUpsert.request` action [gets dispatched](https://github.com/teamdigitale/io-app/blob/master/ts/screens/preferences/ServiceDetailsScreen.tsx#L138), that in turn sets `props.profile` to [an updating Pot](https://github.com/teamdigitale/io-app/blob/master/ts/store/reducers/profile.ts#L50). 

here I think this should be

```ts
pot.isUpdating(this.props.profile)
```

Reply:
once the [API call completes](https://github.com/teamdigitale/io-app/blob/master/ts/sagas/profile.ts#L119), `props.profile` should mutate to [`pot.some`](https://github.com/teamdigitale/io-app/blob/master/ts/store/reducers/profile.ts#L66) or `pot.error`